.yamllint

```diff
@@ -4,4 +4,3 @@ rules:
   truthy: disable
```

Comment:
Do we have a lot of violations of this?

Reply:
All of our GitHub Actions workflow scripts use the `on` keyword, and we cannot replace it with `true`. Therefore, I've disabled that specific check.

Reply:
Because the GitHub Actions parser is looking for `on` and not `true`? Huh.

Reply:
That's right..